src/content/docs/en/guides/astro-db.mdx

```diff
@@ -246,6 +424,33 @@ export const DELETE: APIRoute = async (ctx) => {
 }
 ```
 
+The [Actions feature](/en/guides/actions/) provides a more straightforward approach for inserting data into Astro DB. Here's an example of how to perform a database insert using Actions:
+
+```ts
+// src/actions/index.ts
+import { db, Comment } from 'astro:db';
+import { defineAction } from 'astro:actions';
+import { z } from 'astro:schema';
+
+export const server = {
+  addComment: defineAction({
+    // Actions include type safety with Zod, removing the need
+    // to check if typeof {value} === 'string' in your pages
+    input: z.object({
+      author: z.string(),
+      content: z.string(),
```

Comment:
The rest of the page uses body for the comments while these few examples use content

I suggest making everything uniform

```suggestion
      body: z.string(),
```